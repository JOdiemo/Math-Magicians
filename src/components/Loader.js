const Loader = () => (
  <div className="quote-container loader-container">
    <div className="lds-spinner">
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
);

export default Loader;
