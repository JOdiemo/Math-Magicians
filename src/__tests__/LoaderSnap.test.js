import React from 'react';
import renderer from 'react-test-renderer';
// import { BrowserRouter as Router } from 'react-router-dom';
import Loader from '../components/Loader';

describe('Loader component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Loader />);
    expect(tree).toMatchSnapshot();
  });
});
