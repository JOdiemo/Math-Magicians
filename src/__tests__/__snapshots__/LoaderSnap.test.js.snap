// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component matches the snapshot 1`] = `
<div
  className="quote-container loader-container"
>
  <div
    className="lds-spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;

exports[`Navbar component matches the snapshot 1`] = `
<div
  className="quote-container loader-container"
>
  <div
    className="lds-spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
